app/views/trails/show.json.jbuilder

```diff
@@ -8,4 +8,11 @@ json.distance @trail.distance
 json.duration @trail.duration
 json.food_supply @trail.food_supply
 json.warning @trail.warning
-json.photo_urls @trail.photo_urls
\ No newline at end of file
+json.photo_urls @trail.photo_urls
+json.start_longitude @trail.start_longitude
```

Comment:
I looked at the rails console, has it been changed?
If so, I will amend the variables...
On 1 Mar 2015 13:19, "Clement Choi" <notifications@github.com> wrote:

> In app/views/trails/show.json.jbuilder
> <https://github.com/cchchoi1986/hikehk/pull/65#discussion_r25564181>:
>
> > \ No newline at end of file
> > +json.photo_urls @trail.photo_urls
> > +json.start_longitude @trail.start_longitude
>
> these should be
> json.start_coordinates @trail <https://github.com/trail>.start_coordinates
> json.end_coordinates @trail.end_coordinates
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/65/files#r25564181>.
>